### DISCUSSION TITLE
How to create multivolume ZIP files

### DISCUSSION THREAD

**Discussion by adyster** [gildas-lormeau/zip.js#530] [Q&A] [answered]
I'm having trouble creating multivolume ZIP files. I haven't seen any example code anywhere. 

How do I use the SplitDataWriter? I don't know what to pass to the writerGen object in the constructor. 

Ive tried passing a BlobWriter but it then throws an error about missing next() method.

EDIT: I found code in the test-split-zip.js test case. I was able to create multiple zip files. However, is there a naming convention when I save it? I get an error when trying to extract the files with 7zip.

**Comment by gildas-lormeau** [comment, marked as answer]
The last file should have a "zip" extension. The first files should be named ".z01", ".z02", etc.

**Reply by adyster** [reply]
Awesome, that worked. Thanks!
